src/analytics/script/integration/component/transcription/transcription-selector.js

```diff
@@ -5,7 +5,7 @@ const COLLAPSE = api.internals.ns.selector('collapse');
 
 export const TranscriptionSelector = {
   TRANSCRIPTION: TRANSCRIPTION,
-  COLLAPSE: `${TRANSCRIPTION} > ${COLLAPSE}, ${TRANSCRIPTION} > *:not(${TRANSCRIPTION}, ${COLLAPSE}) > ${COLLAPSE}, ${TRANSCRIPTION} > *:not(${TRANSCRIPTION}, ${COLLAPSE}) > *:not(${TRANSCRIPTION}, ${COLLAPSE}) > ${COLLAPSE}`,
+  COLLAPSE: `${TRANSCRIPTION} > ${COLLAPSE}, ${TRANSCRIPTION} > *:not(${TRANSCRIPTION}:not(${COLLAPSE}) > ${COLLAPSE}, ${TRANSCRIPTION} > *:not(${TRANSCRIPTION}):not(${COLLAPSE}) > *:not(${TRANSCRIPTION}):not(${COLLAPSE}) > ${COLLAPSE}`,
```

Comment:
bonjour, je crois qu'il manque une `)` juste avant `:not` sur le premiere sélecteur corrigé : 

`*:not(${TRANSCRIPTION}):not(${COLLAPSE})`
à la place de
`*:not(${TRANSCRIPTION}:not(${COLLAPSE})`